deploy/helm-chart/README.md

```diff
@@ -1,5 +1,24 @@
 # NGINX Gateway Fabric Helm Chart
 
+- [NGINX Gateway Fabric Helm Chart](#nginx-gateway-fabric-helm-chart)
+  - [Introduction](#introduction)
+  - [Prerequisites](#prerequisites)
+    - [Installing the Gateway API resources](#installing-the-gateway-api-resources)
+  - [Installing the Chart](#installing-the-chart)
+    - [Installing the Chart from the OCI Registry](#installing-the-chart-from-the-oci-registry)
+    - [Installing the Chart via Sources](#installing-the-chart-via-sources)
+      - [Pulling the Chart](#pulling-the-chart)
+      - [Installing the Chart](#installing-the-chart-1)
```

Comment:
What does the 1 do here?

Reply:
I'm not sure, I used a plugin to generate the TOC 🤔

Reply:
Ah right